contracts/NFTMarket.sol

```diff
@@ -154,18 +159,18 @@ contract NFTMarket is IERC721Receiver, ReentrancyGuard {
         // There must be at least 1 recipient.
         if (_recipients.length <= 0) revert NoRecipients();
 
-        // Add up all `SaleRecipeint.percent` allotments to make sure they sum to no more than 100%.
+        // Storage for recipient percentage validation.
         uint256 saleRecipientsPercentTotal = 0;
 
-        // Validate that recipients add up to no more than 100%.
+        // Validate that all `SaleRecipeint.percent` sum to no more than 10000 (100%).
         for (uint256 i = 0; i < _recipients.length; i++) {
             // Each recipient's percent must be greater than 0.
             if (_recipients[i].percent <= 0) revert RecipientPercentZero();
```

Comment:
Just FYI for errors we've been rolling with all caps snake case: RECIPIENT_PERCENT_ZERO()

Reply:
Fixed.